Samsung/A40/res/values/notch.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+<dimen name="status_bar_height_portrait">72px</dimen>
+<dimen name="status_bar_height_landscape">72px</dimen>
+<string name="config_mainBuiltInDisplayCutout"/>
```

Comment:
Well, Galaxy A40 *does* have display cutout.

Though i'm guessing waterdrop hurts pretty much noone, so we don't care, and it looks like Samsung didn't care as well, else this wouldn't be empty.

@TadiT7 what's you're recommendation there?

Reply:
I'm lost here 😁

Reply:
This should be fixed with the latest commit (new values retrieved from framework-res__auto_generated_rro.apk)